examples/api_test.sh

```diff
@@ -42,4 +40,4 @@ sleep 1
 run com.endlessm.Clippy.Emit delete-from-cursor nothing "<('search_entry', 0, 2)>"
 
 sleep 2
-run org.gtk.Actions.Activate 'quit' [] {}
\ No newline at end of file
+run org.gtk.Actions.Activate 'quit' [] {}
```

Comment:
(Commenting here because Github doesn't allow comments on the commit message)
There's a period sign in the first line which is not common, and none in the body. Also, it starts with a capital letter after the module name, but the other commits start with a lowercase letter (I prefer using a capital letter, but it's your module :smile: ).